addons/event/views/event_views.xml

```diff
@@ -349,15 +350,15 @@
                                             <div><i class="fa fa-map-marker" title="Location"/> <span class="o_text_overflow o_event_kanban_location" t-esc="record.address_id.value"/></div>
                                         </div>
                                         <h5 class="o_event_fontsize_11 p-0">
-                                            <a name="%(act_event_registration_from_event)d" type="action">
+                                            <a name="%(act_event_registration_from_event)d" type="action" context="{'search_default_expected': True}">
                                                 <t t-esc="record.seats_expected.raw_value"/> Expected attendees
                                             </a>
                                             <t t-set="total_seats" t-value="record.seats_reserved.raw_value + record.seats_used.raw_value"/>
-                                            <t t-if="total_seats > 0 "><br/>
-                                                <a class="pl-2" name="%(event_event_action_pivot)d" type="action">
+                                            <div class="o_event_kanban_confirmed_mobile" t-if="total_seats > 0 ">
```

Comment:
`pt-2 pt-md-0` seems to achieve the same result and avoids custom css.

Reply:
Forgot that I can use conditional size on padding Bootstrap classes. Thanks Again :)